js/app.js

```diff
@@ -4,3 +4,30 @@
 console.log('App Started');
 console.log('Fruit count', fruits.length);
 console.log('Veggie count', vegetables.length);
+
+var myApp = angular.module('FandV', []);
+
+myApp.controller('appCtrl', ['$scope', function($scope){
+  $scope.fruits = [];
+  $scope.vegetables = [];
+  $scope.both = fruits.concat(vegetables);
+  $scope.click = function(destination, idx, currentArray){
+    $scope[destination].push($scope[currentArray][idx]);
+    $scope[currentArray].splice(idx,1);
+    
+  }
+
+  $scope.checkCorrect = function(currentItem, currentArray){
+    if(currentArray === 'fruits'){
+      if(fruits.indexOf(currentItem) === -1){
+        return true;
+      }
+    } else {
+      if(currentArray === 'vegetables'){
+        if(vegetables.indexOf(currentItem) === -1){
+          return true;
+        }
+      }
+    }
+  }
+}]);
```

Comment:
here is my win check function:
    
    $scope.winCheck = function(){
        
        for (var i = 0; i < $scope.allFruit.length; i++) {
            if ($scope.originalFruit.indexOf($scope.allFruit[i]) == -1) {
                $scope.fail = true;
                return false;
            }
        }
        for (var i = 0; i < $scope.allVegetables.length; i++) {
            if ($scope.originalVegetables.indexOf($scope.allVegetables[i]) == -1) {
                $scope.fail = true;
                return false;
            }
        }
        $scope.win = true;
        console.log($scope.win);
    }